sleap/io/format/csv.py

```diff
@@ -0,0 +1,75 @@
+"""
+Adaptor for writing SLEAP analysis as csv.
+
+"""
```

Comment:
Nit: google style docstrings

```suggestion
"""Adaptor for writing SLEAP analysis as csv."""
```